contrib/inventory/ec2.py

```diff
@@ -129,6 +129,7 @@
 from boto import rds
 from boto import elasticache
 from boto import route53
+import boto3
```

Comment:
@Jonnymcc rds2 ony allows for instance describe, not cluster. http://boto.cloudhackers.com/en/latest/ref/rds2.html#boto.rds2.layer1.RDSConnection.describe_db_instances